internal/metamorph/zmq.go

```diff
@@ -92,11 +91,6 @@ func (z *ZMQ) Start(zmqi ZMQI) error {
 	const invalidTxTopic = "invalidtx"
 	const discardedFromMempoolTopic = "discardedfrommempool"
 	go func() {
-		defer func() {
```

Comment:
Sorry, I missed it. Why don't we need these logs anymore?

Reply:
We simply tried to capture stack trace and log it FROM THE CODE. and we have 24 recovers. It's better to simply capture stderr if the application panics and log it from there.

Reply:
Thanks